src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -243,9 +243,6 @@ export const routes: Routes = {
           UPDATE: {
             path: `/my-contents/videos/:videoId`,
           },
-          INVITE: {
-            path: `/my-contents/videos/:videoId/invite/:inviteId`,
-          },
```

Comment:
Should this be just deleted ?

Reply:
Apparently for the moment we will not use invitation link for the video.